allure-cucumber-jvm/src/main/java/io/qameta/allure/cucumberjvm/AllureCucumberJvm.java

```diff
@@ -134,7 +134,8 @@ public void match(final Match match) {
     @Override
     public void result(final Result result) {
         if (!isNullMatch) {
-            final StatusDetails statusDetails = new StatusDetails();
+            final StatusDetails statusDetails =
+                    ResultsUtils.getStatusDetails(result.getError()).orElse(new StatusDetails());
```

Comment:
It is needed to manage it later. Look at two strings down, I'm adding flacky/muted/known marks to it.